File: QLNH_API/QLNH_Client/Models/Price.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace QLNH_Client.Models
{
    public class Price
    {
        [Key]
        public int Id { get; set; }
        public int? RestaurantId { get; set; }

        public int? ItemId { get; set; }

        public int? SizeId { get; set; }

        public int? UnitId { get; set; }

        [ForeignKey("RestaurantId")]
        public virtual Restaurant? Restaurant { get; set; }

        [ForeignKey("ItemId")]
        public virtual Item? Item { get; set; }

        [ForeignKey("SizeId")]
        public virtual Size? Size { get; set; }

        [ForeignKey("UnitId")]
        public virtual Unit? Unit { get; set; }
        public string Description { get; set; }
        public double SalePrice { get; set; }
        public DateTime Created { get; set; } 
        public DateTime Updated { get; set; }
        public bool Deleted { get; set; } = false;
    }
}

File: QLNH_Angular/QLNH_Angular/src/app/_models/price.ts
import { Item } from "./item";
import { Restaurant } from "./restaurant";
import { Size } from "./size";
import { Unit } from "./unit";

export interface Price{
    id: number,
    salePrice: number,
    description: string,
    created: Date,
    updated: Date,
    deleted: boolean,
    restaurantId: number,
    restaurant: Restaurant,
    unitId: number,
    unit: Unit,
    sizeId: number,
    size: Size,
    itemId: number,
    items: Item
}